pkg/storageincentives/redistributionstate_test.go

```diff
@@ -85,6 +91,76 @@ func TestState(t *testing.T) {
 
 }
 
+func TestStateRoundData(t *testing.T) {
+	t.Parallel()
+
+	t.Run("sample data", func(t *testing.T) {
+		t.Parallel()
+
+		state := createRedistribution(t, nil, nil)
+
+		_, exists := state.SampleData(1)
+		if exists {
+			t.Error("should not exists")
+		}
+
+		savedSample := SampleData{
+			ReserveSample: storage.Sample{
+				Hash: swarm.RandAddress(t),
+			},
+			StorageRadius: 3,
+		}
+		state.SetSampleData(1, savedSample)
+
+		sample, exists := state.SampleData(1)
+		if !exists {
+			t.Error("should exist")
+		}
+		if !reflect.DeepEqual(savedSample, sample) {
```

Comment:
I'd recommend using `cmp.Diff` so there is more insight into why the test failed. The same is below.

Reply:
fixed